tools/clang/test/hlsl_cmds_test/NonUniform.test

```diff
@@ -0,0 +1,56 @@
+// RUN: %dxc -E main -T ps_6_0 %s | FileCheck %s
+
+// CHECK: createHandle
+// CHECK: i1 true)
+
+// CHECK: createHandle
+// CHECK: i1 true)
+
+// CHECK: createHandle
+// CHECK: i1 true)
+
+// RUN: %dxc %s /T ps_6_0 /DDX12 /Fo %t.NonUniform.cso
+// RUN: %dxc %t.NonUniform.cso /dumpbin /Qstrip_rootsignature /Fo %t.NonUniformNoRootSig.cso
+// RUN:%dxa  -listparts %t.NonUniformNoRootSig.cso | FileCheck %s --check-prefix=NO_RS
+
+// NO_RS:Part count
+// NO_RS-NOT:RTS0
+
+// RUN: %dxc  %t.NonUniform.cso /dumpbin /extractrootsignature /Fo %t.NonUniformRootSig.cso
+// RUN:%dxa  -listparts %t.NonUniformRootSig.cso | FileCheck %s --check-prefix=RS_PART
```

Comment:
We don't build dxa on Linux at the moment, so we should look at the tests that use it and evaluate if it makes sense to support dxa on Linux, or if we should adjust the test or only run the test on Windows.